cmd/registry/provider_test.go

```diff
@@ -0,0 +1,48 @@
+package registry
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/astronomer/astro-cli/pkg/fileutil"
+	testUtil "github.com/astronomer/astro-cli/pkg/testing"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+func execProviderCmd(args ...string) (string, error) {
+	buf := new(bytes.Buffer)
+	cmd := newRegistryProviderCmd(os.Stdout)
+	cmd.SetOut(buf)
+	log.SetOutput(buf)
+	defer func() {
+		log.SetOutput(os.Stderr)
+	}()
+	cmd.SetArgs(args)
+	testUtil.SetupOSArgsForGinkgo()
+	_, err := cmd.ExecuteC()
+	return buf.String(), err
+}
+
+func TestProviderAdd(t *testing.T) {
+	testUtil.InitTestConfig(testUtil.CloudPlatform)
+
+	_ = os.Remove("requirements.txt")
```

Comment:
```suggestion
	defer os.Remove("requirements.txt")
```
nit: simplify code so that we don't need multiple `os.Remove`